exec/vector/src/main/java/org/apache/drill/exec/vector/accessor/impl/VectorPrinter.java

```diff
@@ -33,7 +32,10 @@
   public static void printOffsets(UInt4Vector vector, int start, int length) {
     header(vector, start, length);
     for (int i = start, j = 0; j < length; i++, j++) {
-      if (j > 0) {
+      if (j % 40 == 0) {
```

Comment:
Before this change, I had a vector of 1000 items all on one line. After this change, the output is 40 elements per line. Note that this code is used only during debugging.